packages/core/src/theme/Theme.ts

```diff
@@ -28,25 +28,26 @@ export class Theme {
     this.id = id;
     this.name = name;
   }
+}
 
-  getCharacteristicValue(
-    characteristicName: characteristic,
-    variant: string,
-    scopeElement?: HTMLElement
-  ): string | null {
-    const cssVariableName = `--uitk-${characteristicName}-${variant}`;
-    const scopeTarget =
-      scopeElement || document.body.querySelector(`.uitk-${this.name}`);
-    if (scopeTarget) {
-      const style = getComputedStyle(scopeTarget);
-      const variableValue = style.getPropertyValue(cssVariableName);
-      if (variableValue) {
-        return variableValue;
-      }
+export const getCharacteristicValue = (
+  themeName: string,
+  characteristicName: characteristic,
+  variant: string,
+  scopeElement?: HTMLElement
+): string | null => {
+  const cssVariableName = `--uitk-${characteristicName}-${variant}`;
+  const scopeTarget =
+    scopeElement || document.body.querySelector(`.${themeName}`);
+  if (scopeTarget) {
+    const style = getComputedStyle(scopeTarget);
+    const variableValue = style.getPropertyValue(cssVariableName);
+    if (variableValue) {
+      return variableValue;
     }
-    return null;
   }
-}
+  return null;
+};
 
 const lightTheme = new Theme("uitk-light", "light");
 const darkTheme = new Theme("uitk-dark", "dark");
```

Comment:
I believe we should call these modes.
I feel we might need to change few things in here also, as we now have only one theme and two extra modes...??

Reply:
I actually think we should consider removing all the code from here to the end of the file